pdns/README-dnsdist.md

```diff
@@ -1425,6 +1438,7 @@ instantiate a server with additional parameters
     * `clearDynBlocks()`: clear all dynamic blocks
     * `showDynBlocks()`: show dynamic blocks in force
     * `addDynBlocks(addresses, message[, seconds])`: block the set of addresses with message `msg`, for `seconds` seconds (10 by default)
+    * `addWhitelistedDynBlocks(addresses, message[, seconds[, whitelist]])`: block the set of addresses with message `msg`, for `seconds` seconds. Addresses present in the`whitelist` `NetmaskGroup` will not be blocked even if they match
```

Comment:
Any reason you didn't just add another argument to `addDynBlocks`? I'm not sure about the naming of the new function either, maybe `addDynBlocksWithWhitelist` but I feel I'm channeling too much ObjC here ;)